pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line based tool to make presentations from markdown formatted files.
+
+- Start a presentation from a markdown file:
+
+`mdp {{presentation.md}}`
+
+- Disable fading transitions:
+
+`mdp --nofade {{presentation.md}}`
+
+- Invert colors for light background:
```

Comment:
This assumes your terminal is set to a dark background, right? Or does it always use a dark background by default regardless of the terminal's color scheme?